src/nodes/ImageView.vue

```diff
@@ -257,6 +256,12 @@ export default {
 				})
 			},
 		},
+		fileId() {
+			return getQueryVariable(this.src, 'fileId')
+		},
+		mimeType() {
+			return getQueryVariable(this.src, 'mimeType')
+		},
```

Comment:
This change is not required either if I'm not mistaken. Really just add the `a` tag above and that's it, no?

Reply:
@mejo- It is required to detect if this file type is supported or not.